src/EditorFeatures/Core/IntelliSense/AsyncCompletion/CommitManagerProvider.cs

```diff
@@ -30,7 +30,7 @@ internal class CommitManagerProvider : IAsyncCompletionCommitManagerProvider
             IThreadingContext threadingContext,
             RecentItemsManager recentItemsManager,
             IGlobalOptionService globalOptions,
-            [Import(AllowDefault = true)] ILanguageServerSnippetExpander? languageServerSnippetExpander)
+            ILanguageServerSnippetExpander? languageServerSnippetExpander)
```

Comment:
This might not be OK for VS4Mac, the only implementation I found is in WPF layer.